README.md

```diff
@@ -334,6 +335,12 @@ If you are writing your blog post in markdown, save your `.md` file into the `/_
 
 Save your Microsoft Word documents into the `/_word` folder with the same naming convention (`YYYY-MM-DD-*.docx`) specified for notebooks.
 
+### Specifying front-matter for Word documents
```

Comment:
Since we are adding some instructions for creating documents, can we add a note that pictures shouldnt have any `alt text` either as something that user should be aware but not necessary as a instruction

Reply:
I'm weary of putting this in the instructions as there could be many, many edge cases and this is only one of N that we have found.  Perhaps an FAQ, but not sure

Reply:
eh.  I just added it

Reply:
That is true. Dealing with word document conversion is really tough. I am glad that there is basic support in fastpages